README.md

```diff
@@ -1 +1,23 @@
-# COVID_moonshot_data_clean
\ No newline at end of file
+# COVID_moonshot_data_clean
+This repository contains cleaned versions of the data in
+https://github.com/asapdiscovery/COVID_moonshot_submissions.
+
+## Files
+* `cdd_noncovalent_dates_2023_10_18_unfiltered_clean.csv`: All noncovalent available
+data as of October 10, 2023, cleaned to remove data without a noted SMILES string or
+IC50 value
+* `cdd_achiral_enantiopure_dates_2023_10_18_filt.csv`: All noncovalent available data as
+of October 10, 2023, filtered to contain only achiral and enantiopure molecules.
+
+## Column Headers
+A brief explanation of each column present in the CSV files.
+* `Canonical_PostEra_ID`: Molecule's canonical name
+* `suspected_SMILES`: SMILES string of the synthesized molecule
```

Comment:
is there somewhere we can refer to for a clarification on this? I imagine externals might not immediately understand the difference between 'regular' and 'suspected' SMILES.

Reply:
would it be better if I just renamed the field to `SMILES`? or would the be obfuscating too much?

Reply:
you can just post this in the readme, copied from https://github.com/postera-ai/COVID_moonshot_submissions#details-on-smiles-and-molecule-identities:

On `suspected_SMILES`: Note that many of the enantiopure compounds on this project were obtained by chiral separation, and thus the compounds are often obtained as a single enantiomer with unknown absolute stereochemistry. The `suspected_SMILES` column represents the understanding of the actual identity of the compound given the current information, which explains the frequent use of enhanced/relative stereochemistry representations. Keep in mind that the `suspected_SMILES` is often not the same as the `registration_SMILES`, since compounds must be registered as a real (non-relatively defined) compound on this project.

Reply:
sounds good, done in https://github.com/asapdiscovery/COVID_moonshot_data_clean/pull/1/commits/aabc260f0485cb95d2a957f4faa91e1651a9145f

Reply:
In addition to adding this note to the README, I would keep `suspected_SMILES` as the first column, PostEra ID in the second column, and `registration_SMILES` as another column.